tests/test_channels.py

```diff
@@ -0,0 +1,110 @@
+from lib import sdk
+
+import json
+
+
+s = sdk.SDK()
+
+channel = {"channel_name": "channel"}
+channel_id = "123-456-789"
+channel_id1 = "123-223-333"
+thing_id = "332-665-998"
+token = "9a8b7c6d5e4f3g21"
+url = "http://localhost"
+params = None
+
```

Comment:
No need to skip 2 lines, and this should be corrected for the whole file

Reply:
There should be two blank lines.